frontend/config-assistant/src/components/toolbar/clearFile.ts

```diff
@@ -2,8 +2,13 @@ import {ChangeResponsible, useSessionStore} from '@/store/sessionStore';
 import {toastService} from '@/helpers/toastService';
 import {confirmationService} from '@/helpers/confirmationService';
 import {useDataStore} from '@/store/dataStore';
+import _ from 'lodash';
 
 export function newEmptyFile(message: string | undefined = undefined): void {
+  if (_.isEmpty(useDataStore().fileData)) {
```

Comment:
what happens if you click on new file in schema editor while file editor is empty?

Reply:
it will ask for clearing the schema and if accepted will clear the schema. It will do nothing with the file editor.
For clearing the schema, there is a separate toolbar/clearSchema.ts file.